binding.gyp

```diff
@@ -47,6 +47,7 @@
 					],
 					"xcode_settings": {
 						"OTHER_LDFLAGS": [
+							"-fvisibility=hidden",
```

Comment:
I'm not sure if it's best to add the flag under OTHER_LDFLAGS or ldflags. Or if they are equivalent...